external/model-preparation-algorithm/mpa_tasks/apis/classification/task.py

```diff
@@ -381,10 +375,13 @@ def _patch_evaluation(self, config: MPAConfig):
         cfg = config.evaluation
         if self._multilabel:
             cfg.metric = ['accuracy-mlc', 'mAP', 'CP', 'OP', 'CR', 'OR', 'CF1', 'OF1']
+            config.early_stop_metric = 'mAP'
         elif self._hierarchical:
             cfg.metric = ['MHAcc', 'avgClsAcc', 'mAP']
+            config.early_stop_metric = 'mAP'
```

Comment:
Could you change into `MHAcc` or `accuracy`? mAP can be always 0.0 if hierarchical project is consisted only with single-label classification groups.
For 'accuracy', please refer this PR : https://github.com/openvinotoolkit/training_extensions/pull/1215